packages/core-utils/src/io-utils.test.ts

```diff
@@ -0,0 +1,14 @@
+import { readFileAsText } from './io-utils';
+
+describe('readFileAsText', () => {
+  it('returns file contents', async () => {
+    // Arrange
+    const file = new File(['Hello world'], 'test-file.txt');
+
+    // Act
+    const content = await readFileAsText(file);
+
+    // Assert
+    expect(content).toEqual('Hello world');
+  });
```

Comment:
a test case for thrown error would be great for completeness.

Reply:
I had skipped this because it seemed like it would require so much mocking that it wouldn't really be useful.

But after looking into it, now I'm thinking that this whole function should be deleted and replaced with `File.text`.
https://developer.mozilla.org/en-US/docs/Web/API/File#instance_methods

Reply:
Never mind. Browser support for `Blob.text` doesn't match what we have listed in `outdated-browser-message` (which I'm taking as what we "officially" support).

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4216/commits/92ca64caa617507be8988004d319bb1606b2689a.

Reply:
nice!